lm-agent/lm_agent/config.py

```diff
@@ -10,7 +10,7 @@
 from lm_agent.constants import LogLevelEnum
 
 
-DEFAULT_DOTENV_PATH = Path("/etc/default/license-manager-agent")
+DEFAULT_DOTENV_PATH = Path("/var/snap/license-manager-agent/common/.env")
```

Comment:
Is this change really necessary?
In Scania environment the .env file is loaded from the default path, the charm is configured to populate the env vars there.

Reply:
this is needed according to our intention in making the agent installable by snap instead of configuring the virtual environment, installing the python package, etc...

Reply:
Damn...Juliana is right, this will cause issues for our deployments of lm-agent on non-Ubuntu machines. We have some deployments on Rocky 9 and those environments do not have snap installed (and we will not be able to install it, either).

Instead of having one path we look in for the dotenv file, perhaps we need to have a set of places that we look in order until we find a dotenv file. What do you both think of this?

Reply:
we discussed in private that we would move with this implementation. Can we resolve @dusktreader @julianaklulo ?

Reply:
As per our prior discussion, the charm-license-manager-agent is already configuring the systemd service to use an `EnvironmentFile`. We verified that agent when deployed with the charm is able to load its config successfully without needing to set the DEFAULT_DOTENV_PATH to `/etc/default/license-manager-agent`. Resolving this comment since we are moving forward as planned and tested.